packages/fx-core/src/component/driver/aad/create.ts

```diff
@@ -79,14 +80,30 @@ export class CreateAadAppDriver implements StepDriver {
           )
         );
         context.addTelemetryProperties({ [telemetryKeys.newAadApp]: "true" });
+
+        const tokenJson = await context.m365TokenProvider.getJsonObject({ scopes: GraphScopes });
+        const isMsftAccount =
+          tokenJson.isOk() &&
+          tokenJson.value.unique_name &&
+          (tokenJson.value.unique_name as string).includes("@microsoft.com");
```

Comment:
it's better to use endsWith("@microsoft.com")?

Reply:
Good catch! @xiaolang124 do you have any concerns of using `endsWith("@microsoft.com")` to determine microsoft account? If not, I will make the change.